cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -442,7 +442,7 @@ impl<'flags, I: VCodeInst> VCode<'flags, I> {
 
     /// Mutate branch instructions to (i) lower two-way condbrs to one-way,
     /// depending on fallthrough; and (ii) use concrete offsets.
-    pub fn finalize_branches(&mut self)
+    pub fn finalize_branches(&mut self, flags: &settings::Flags)
```

Comment:
Given that #1530 will land first, can't we use the flags held by the `VCode` directly here (i.e. `self.flags`)?